web/modules/custom/dpl_react_apps/dpl_react_apps.module

```diff
@@ -213,7 +219,13 @@ function dpl_react_apps_texts(): array {
     'publizon-podcast' => t('Podcast', [], ['context' => 'Global']),
     'ready-for-loan-counter-label' => t('Ready', [], ['context' => 'Global']),
     'ready-for-loan' => t('Ready for pickup', [], ['context' => 'Global']),
-    'remove-all-reservations' => t('Remove reservations (@amount)', [], ['context' => 'Global']),
+    'remove-all-reservations' => json_encode([
+      'type' => 'plural',
+      'text' => [
+        t('Remove reservation (@amount)', [], ['context' => 'Global']),
```

Comment:
Since the amount will always be "1" in the singular form, perhaps we should consider omitting the amount?
```suggestion
        t('Remove reservation', [], ['context' => 'Global']),
```

Reply:
I see your point, but we provide this amount variable from the react apps when calling the `t()` function, so I think I'll just keep it in :)

Reply:
Cool :)